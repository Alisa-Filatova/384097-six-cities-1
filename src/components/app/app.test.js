import React from 'react';
import renderer from 'react-test-renderer';
import App from './app.jsx';

const mock = [
  {
    id: 123,
    title: `Beautiful & luxurious apartment at great location`,
    img: `img/apartment-01.jpg`,
    isPremium: true,
    price: 120,
    stars: 4,
    type: `Apartment`,
    isInBookmarks: false,
  },
];

describe(`App`, () => {
  it(`renders correctly`, () => {
    const app = renderer.create(
        <App rentalOffers={mock} />
    ).toJSON();

    expect(app).toMatchSnapshot();
  });
});
