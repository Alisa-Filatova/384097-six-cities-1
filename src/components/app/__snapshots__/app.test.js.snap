// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<Fragment>
  <PageWrapper>
    <AppHeader
      isAuthenticated={true}
      user={
        Object {
          "avatarUrl": "/static/avatar/4.jpg",
          "email": "alicefill@y888.ru",
          "id": 1,
          "isPro": false,
          "name": "alicefill",
        }
      }
    />
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        path="/onLogin"
      />
      <Route
        component={
          Object {
            "$$typeof": Symbol(react.memo),
            "WrappedComponent": [Function],
            "compare": null,
            "displayName": "Connect(OfferDetails)",
            "type": [Function],
          }
        }
        path="/offer/:id"
      />
      <Route
        component={[Function]}
        path="/favorites"
      />
      <Route
        component={[Function]}
        path="/error"
      />
      <Route
        component={[Function]}
      />
    </Switch>
  </PageWrapper>
</Fragment>
`;
